code/game/gamemodes/changeling/powers/mutations.dm

```diff
@@ -126,7 +126,7 @@
 	icon = 'icons/obj/weapons.dmi'
 	icon_state = "arm_blade"
 	item_state = "arm_blade"
-	flags = ABSTRACT | NODROP
+	flags = NODROP

```

Comment:
Wait, arm_blade was already abstract but shield wasn't? In that case, I stand corrected It's probably easier to give the shield the same flags, even if it's a bit odd.